.gitignore

```diff
@@ -40,4 +40,8 @@ not_for_github/auto_scorer/sheet_to_local/*
 #RDB stuff
 win-RDB-venv/
 ltm_agent.db
-.vscode
\ No newline at end of file
+.vscode
+
+#Eval Stuff
+retrieval_evaluator/comparison_data/*
+retrieval_evaluator/evaluation_outputs/*
```

Comment:
This could probably go under `data`, which is ignored by default.

Reply:
Moved data related to data and the script under Utils